content/en/opentelemetry/guide/combining_otel_and_datadog_metrics.md

```diff
@@ -15,6 +15,8 @@ The <code>equiv_otel()</code> function is in Preview. If you have feedback relat
 
 Datadog and OpenTelemetry (OTel) use different naming conventions for integration metrics. This guide explains how to combine metrics from both systems in a single query using Datadog's `equiv_otel` function.
 
+<div class="alert alert-info">To query across Datadog and OpenTelemetry metrics in the Datadog UI, read the <a href="/metrics/open_telemetry/query_metrics">Metrics</a> documentation.</div>
```

Comment:
```suggestion
<div class="alert alert-info">To query across Datadog and OpenTelemetry metrics in the Datadog UI, read the <a href="/metrics/open_telemetry/query_metrics">Query OpenTelemetry Metrics</a> documentation.</div>
```

also just a suggestion, but similar reasoning re: informative link text that doesn't make users feel like they're about to jump to a high level of generality